### PR TITLE
Background home

### PR DESCRIPTION
**Pull request by quandysse**
Home finish

### CODE REVIEW

File: portfolio.html [hunk 1]
```diff
@@ -2,12 +2,44 @@
 <html>
 <head>
     <link rel="stylesheet" href="style.css">
-    <title>Portfolio</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Palanquin+Dark:wght@600&family=Palanquin:wght@100;300;500;600&family=Raleway:wght@100&family=Roboto:wght@300&display=swap" rel="stylesheet">
+    <title>Candice Test</title>
```
Comment (bobylegros) [inline portfolio.html:7]: Indentation PB

File: portfolio.html [hunk 2]
```diff
@@ -2,12 +2,44 @@
...
 </head>
 <body>
-
-<h1>My First Heading</h1>
-<p>My first paragraph.</p>
-
+    <div class="section1">
```
Comment (bobylegros) [inline portfolio.html:10]: You could use the <section> balise

File: portfolio.html [hunk 3]
```diff
@@ -2,12 +2,44 @@
...
+            <a href="#">Dessin</a>
+            <a href="#">Photo</a>
+            <div class="animation start-home"></div>
+        </nav>
+    </div>
+    <div class="section2">
+    
+    </div>
 </body>
 </html>
```
Comment (bobylegros) [inline portfolio.html:45]: Indentation PB

**Review by bobylegros — COMMENTED** [review]
Take a look at your indentation and your tags.

File: portfolio.html (hunk 1 above)
Comment (quandysse) [reply portfolio.html:7]: ok it's clean boby

File: portfolio.html (hunk 2 above)
Comment (quandysse) [reply portfolio.html:10]: ok it's clean

File: portfolio.html (hunk 3 above)
Comment (quandysse) [reply portfolio.html:45]: ok it's clean

**Review by bobylegros — APPROVED** [review]
Ok it's good to merge

**Pull request merged**